composables/useGraphql.ts

```diff
@@ -43,16 +48,24 @@ export default function ({
     }
   }
 
+  async function refetch() {
+    await doFetch({
+      options: {
+        fetchPolicy: 'network-only',
+      },
+    })
+  }
+
   if (isRef(variables)) {
-    watchEffect(doFetch)
+    watchEffect(() => doFetch())
```

Comment:
This is to solve the TS error. Because `watchEffect` passes onCleanup to effect, which is inconsistent with the newly added parameter type of `doFetch`.

Reply:
Nice I learned something today ❤️

https://vuejs.org/api/reactivity-core.html#watcheffect